src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -97,8 +97,10 @@
 import java.nio.charset.Charset;
 import java.nio.charset.UnsupportedCharsetException;
 import java.util.*;
+import java.util.List;
```

Comment:
Hm, compiler requires it. I think this may be related to the * imports, and probably two List types in two * imports requiring the specific import.